presto-docs/src/main/sphinx/develop/presto-page.rst

```diff
@@ -0,0 +1,233 @@
+=================================
+PrestoPage Wire Format
+=================================
+
+Presto uses SerializedPage binary columnar format for exchanging data between
```

Comment:
Presto uses SerializedPage **as** binary columnar

Reply:
I don't think the "as" is needed here.